Source/BugsnagSessionTrackingPayload.m

```diff
@@ -24,33 +26,57 @@ @interface Bugsnag ()
 + (BugsnagClient *)client;
 @end
 
+@interface BugsnagApp ()
++ (BugsnagApp *)appWithDictionary:(NSDictionary *)event config:(BugsnagConfiguration *)config;
+
+- (NSDictionary *)toDict;
+@end
+
+@interface BugsnagSessionTrackingPayload ()
+@property (nonatomic) BugsnagConfiguration *config;
+@end
+
 @implementation BugsnagSessionTrackingPayload
 
-- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions {
+- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions config:(BugsnagConfiguration *)config {
```

Comment:
nit: looks better formatted over three lines (brace on separate line).

Reply:
Moved the brace for now. We should probably decide on a common style at some point down the line, as parts of the codebase are inconsistent on formatting.